src/Umbraco.Infrastructure/Migrations/MigrationPlanExecutor.cs

```diff
@@ -147,14 +145,14 @@ public ExecutedMigrationPlan ExecutePlan(MigrationPlan plan, string fromState)
         // If any completed migration requires us to sign out the user we'll do that.
         if (_invalidateBackofficeUserAccess)
         {
-            RevokeBackofficeTokens().GetAwaiter().GetResult(); // should async all the way up at some point
+            await RevokeBackofficeTokens().ConfigureAwait(false);
         }
 
         return result;
     }
 
     [Obsolete]
-    private void HandlePostMigrations(ExecutedMigrationPlan result)
+    private async Task HandlePostMigrationsAsync(ExecutedMigrationPlan result)
```

Comment:
The post migrations have been obsoleted for a while (even stated for removal in v13, which didn't happen), but removing them should be done in a separate PR.